templates/partials/versions.html

```diff
@@ -0,0 +1,18 @@
+<h2>{{ trans(key='current_versions', lang=lang) }}</h2>
+{{ trans(key='current_versions_text', lang=lang) }}
```

Comment:
```suggestion
```
I think that this line is redundant.

We know that these are the current versions of Vala available from the title of the section.

Reply:
If you insist on having the line there anyway, wrap it with the `<p>` tag.